[rospy.client][INFO] 2020-02-03 23:43:26,689: init_node, name[/rostopic_9241_1580753606644], pid[9241]
[xmlrpc][INFO] 2020-02-03 23:43:26,690: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2020-02-03 23:43:26,690: Started XML-RPC server [http://itachi-HP-Pavilion-Laptop-15-cc1xx:43903/]
[rospy.init][INFO] 2020-02-03 23:43:26,690: ROS Slave URI: [http://itachi-HP-Pavilion-Laptop-15-cc1xx:43903/]
[rospy.impl.masterslave][INFO] 2020-02-03 23:43:26,690: _ready: http://itachi-HP-Pavilion-Laptop-15-cc1xx:43903/
[rospy.registration][INFO] 2020-02-03 23:43:26,691: Registering with master node http://localhost:11311
[xmlrpc][INFO] 2020-02-03 23:43:26,691: xml rpc node: starting XML-RPC server
[rospy.init][INFO] 2020-02-03 23:43:26,790: registered with master
[rospy.rosout][INFO] 2020-02-03 23:43:26,791: initializing /rosout core topic
[rospy.rosout][INFO] 2020-02-03 23:43:26,794: connected to core topic /rosout
[rospy.simtime][INFO] 2020-02-03 23:43:26,796: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rospy.internal][INFO] 2020-02-03 23:43:26,857: topic[/detection_info] adding connection to [http://itachi-HP-Pavilion-Laptop-15-cc1xx:36699/], count 0
[rospy.internal][INFO] 2020-02-03 23:43:27,037: topic[/rosout] adding connection to [/rosout], count 0
[rospy.core][INFO] 2020-02-03 23:43:53,800: signal_shutdown [signal-2]
[rospy.internal][INFO] 2020-02-03 23:43:53,804: topic[/rosout] removing connection to /rosout
[rospy.internal][INFO] 2020-02-03 23:43:53,804: topic[/detection_info] removing connection to http://itachi-HP-Pavilion-Laptop-15-cc1xx:36699/
[rospy.internal][WARNING] 2020-02-03 23:43:53,805: Unknown error initiating TCP/IP socket to itachi-HP-Pavilion-Laptop-15-cc1xx:42427 (http://itachi-HP-Pavilion-Laptop-15-cc1xx:36699/): Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 650, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
AttributeError: 'NoneType' object has no attribute 'buff_size'

[rospy.topics][ERROR] 2020-02-03 23:43:53,805: Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/topics.py", line 326, in close
    c.close()
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 838, in close
    self.socket.close()
AttributeError: 'NoneType' object has no attribute 'close'

[rospy.impl.masterslave][INFO] 2020-02-03 23:43:53,805: signal-2
[rospy.core][INFO] 2020-02-03 23:43:53,938: signal_shutdown [atexit]
